client/question/services/EventHandlerServiceSpec.js

```diff
@@ -58,40 +60,35 @@ describe('EventHandlerService', function() {
 
   describe('createSessionResumeEvent', function() {
 
-    it('sends a POST request to the backend to create a SessionResumeEvent',
+    it('creates a SessionResumeEvent and adds it to the current EventBatch',
       function() {
-        $httpBackend.expectPOST(
-          '/ajax/event/create_session_resume_event').respond(
-          HTTP_STATUS_CODE_OK, {});
         spyOn(ServerHandlerService, 'doesServerExist').and.returnValue(true);
         EventHandlerService.createSessionResumeEvent(sessionId);
-        $httpBackend.flush();
+        expect(EventHandlerService.currentEventBatch.length).toEqual(1);
```

Comment:
Here and below: either check that the length is 0 at the outset, or verify the type of the first event at the end, or both.

Reply:
Done.